includes/wpmenucart-settings.php

```diff
@@ -698,7 +700,7 @@ public function menus_select_element_callback( $args ) {
 			$disabled = ($x == 1) ? '' : ' disabled';
 			
 			$html .= sprintf( '<select name="%1$s[%2$s][%3$s]" id="%1$s[%2$s][%3$s]"%4$s>', esc_attr( $menu ), esc_attr( $id ), esc_attr( $x ), esc_attr( $disabled ) );
-			$html .= sprintf( '<option value="%s"%s>%s</option>', '0', selected( $current, '0', false ), '' );
+			$html .= sprintf( '<option value="%s"%s>%s</option>', '0', selected( $current, '0', false ), __( 'None', 'wp-menu-cart' ) );
```

Comment:
I think you don't need to pass a value for this option, can be empty by default.

Reply:
@alexmigf done: [79c8502](https://github.com/wpovernight/wp-menu-cart/pull/50/commits/79c85026aa517c3b284092fc9108e08f3e97e8a7)